src/views/Institutions/InstitutionComponents/MoUs.jsx

```diff
@@ -1,31 +1,37 @@
-import { useMemo } from "react";
 import Table from "../../../components/content/Table";
-import { Anchor } from "../../../components/content/Utils";
 
 const MoUs = ({ mou }) => {
-  mou = mou.map((mou_file) => {
-    mou_file.mou_file.url = <Anchor text={mou_file.mou_file.url} href={mou_file.mou_file.url} />;
-    return mou_file;
-  });
+  mou = mou.map((mou_obj) => {
+    if (mou_obj.mou_file.hasOwnProperty("url")) {
+      let url = mou_obj.mou_file.url;
+      if (typeof url === "string") {
+        let file_name = url.split("/").pop();
+        mou_obj.mou_file.url = (
+          <a href={url} target="_blank" rel="noopener noreferrer">
+            {file_name}
+          </a>
+        );
+      }
+    } else {
+      mou_obj.mou_file.url = "No Data Found";
+    }
 
-  const columns = useMemo(
-    () => [
-      {
-        Header: 'Name',
-        accessor: 'mou_file.url',
-      },
-      {
-        Header: 'Start Date',
-        accessor: 'start_date',
-      },
-      {
-        Header: 'End Date',
-        accessor: 'end_date',
-      },
-    ],
-    []
-  );
+    const formatDate = (date) => {
+      return date.split("/").reverse().join("/");
+    };
+    mou_obj.start_date = formatDate(mou_obj.start_date);
+    mou_obj.end_date = formatDate(mou_obj.end_date);
 
+    mou_obj.start_date = new Date(mou_obj.start_date).toLocaleDateString();
+    mou_obj.end_date = new Date(mou_obj.end_date).toLocaleDateString();
```

Comment:
the format for the date was mm/dd/yyyy, to change it to dd/m/yyyy I split the format for the first '/' and reversed it and the joined it.